plugins/services/src/js/columns/ServicesTableStatusColumn.tsx

```diff
@@ -3,51 +3,82 @@ import { Trans } from "@lingui/macro";
 import { TextCell } from "@dcos/ui-kit";
 
 import StringUtil from "#SRC/js/utils/StringUtil";
-import ServiceStatusIcon from "../components/ServiceStatusIcon";
+import { SortDirection } from "#PLUGINS/services/src/js/types/SortDirection";
+import ServiceStatusProgressBar from "#PLUGINS/services/src/js/components/ServiceStatusProgressBar";
 import * as ServiceStatus from "../constants/ServiceStatus";
-import ServiceStatusProgressBar from "../components/ServiceStatusProgressBar";
+import ServiceStatusIcon from "../components/ServiceStatusIcon";
 import Pod from "../structs/Pod";
 import Service from "../structs/Service";
-import ServiceTree from "../structs/ServiceTree";
-import { SortDirection } from "plugins/services/src/js/types/SortDirection";
+import ServiceTree, { ServiceTreeStatusSummary } from "../structs/ServiceTree";
 import ServiceTableUtil from "../utils/ServiceTableUtil";
 
-const StatusMapping: any = {
-  Running: "running-state"
-};
+function statusCountsToTooltipContent(
+  statusCounts: Record<ServiceStatus.StatusCategory, number>
+): JSX.Element[] {
+  return Object.keys(statusCounts)
+    .filter(value => value in ServiceStatus.StatusCategory)
+    .sort(statusCategorySorter)
+    .map((value, index) => {
+      const category = value as ServiceStatus.StatusCategory;
+      return (
+        <div key={`status.${index}`}>
+          {statusCounts[category]}{" "}
+          <Trans id={ServiceStatus.toCategoryLabel(category)} />
+        </div>
+      );
+    });
+}
 
 export function statusRenderer(
   service: Service | ServiceTree
 ): React.ReactNode {
-  const serviceStatusText: string = service.getStatus();
-  const serviceStatusClassSet: string = StatusMapping[serviceStatusText] || "";
-  const instancesCount = service.getInstancesCount() as number;
-  const runningInstances = service.getRunningInstancesCount() as number;
-  // L10NTODO: Pluralize
-  const tooltipContent = (
-    <Trans render="span">
-      {runningInstances} {StringUtil.pluralize("instance", runningInstances)}{" "}
-      running out of {instancesCount}
-    </Trans>
-  );
-  const hasStatusText = serviceStatusText !== ServiceStatus.NA.displayName;
+  const serviceStatus:
+    | string
+    | ServiceTreeStatusSummary
+    | null = service.getStatus();
+  let statusContent: JSX.Element | null = null;
+  let tooltipContent: JSX.Element;
+  if (typeof serviceStatus === "object" && serviceStatus !== null) {
+    const statusText = ServiceStatus.toCategoryLabel(serviceStatus.status);
+    statusContent = (
+      <span className="status-bar-text">
+        <Trans id={statusText} />{" "}
+        <Trans id={serviceStatus.countsText} values={serviceStatus.values} />
```

Comment:
I'm not sure this `values` prop is meaningful. What is it?

Reply:
The values for translation of “(x of y)”

Reply:
Now that that’s separate we could do that directly here instead of with an id. I didn’t realize that when refactoring the status out.

In that case values won’t be needed here.